### PR TITLE
Userguide

### PR DESCRIPTION
**Pull request by zhuleyan**
Update dev-guide and user-guide to v1.3

### CODE REVIEW

**Comment by davidten** [comment]
@Sheikh-Umar think you need to resolve this conflict as well

**Comment by davidten** [comment]
@Sheikh-Umar there's more to resolve

**Comment by Sheikh-Umar** [comment]
Conflicts resolved.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -1136,25 +1169,49 @@ Use case resumes at step 2.
...
++
+Use case ends.
+
+=== Use case: User requests to import a CSV file to CRM Book
+
+*MSS*
+
+1.  User enters command for importing file.
+2.  CRM opens the corresponding CSV file.
+3.  CRM processes data and lists all persons including newly added ones.
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: @zhuleyan this does not follow the MSS principle. You need to take turns, so 3 cannot be the CRM again, it has to be the user.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -1219,29 +1276,30 @@ Use case resumes at step 4.
...
-* 3a. The given index is invalid.
+* 2a. Lead entered has a phone number or email address stored earlier.
+** 2a1. CRM Book displays message that a possible duplicate detected.
 +
-** 3a1. CRM Book displays an error message.
+Use case ends.
+
+* 2b. Lead entered does not have a phone number or email address stored earlier.
+** 2b1. CRM Book adds Lead.
+** 2b2. CRM Book displays message of successful addition of Lead.
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: Similarly, MSS

Comment (zhuleyan) [reply docs/DeveloperGuide.adoc]: I've updated my part. I think this section belongs to Woody?

Comment (davidten) [reply docs/DeveloperGuide.adoc]: @zhuleyan But it's in your PR. Can you look through your PR and remove the things that you did not add

Comment (zhuleyan) [reply docs/DeveloperGuide.adoc]: fixed now

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -1194,25 +1227,50 @@ Use case resumes at step 2.
...
 
+*Extensions*
+* 2a. Lead entered has a phone number or email address stored earlier.
+** 2a1. CRM Book displays message that a duplicate is detected.
++
+Use case ends.
+
+* 2b. Lead entered does not have a phone number or email address stored earlier.
+** 2b1. CRM Book adds the Lead.
+** 2b2. CRM Book displays message that the Lead has been added.
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: MSS

Comment (zhuleyan) [reply docs/DeveloperGuide.adoc]: don't understand why it's still there after I removed it.
I think it's ok now

**Pull request merged**